chapters/02-basic-config.md

```diff
@@ -5,6 +5,24 @@ Setup your editor in a way that'll be nice for you to look at and it won't be fi
 I recommend to create your own configuration but it takes time and effort. \
 We will talk about the advantages at the [advanced configuration chapter](https://github.com/ofirgall/learn-nvim/blob/master/chapters/08-advanced-config.md) but I recommend to use preconfigured configuration for now, unless you are a config nerd like me.
 
+---
+
+## Terminal Emulator Setup
+nvim can be run with a GUI frontend or how it was intended, inside a terminal emulator. \
+I recommend to use one of the modern cross-platform terminal emulators:
+
+* [Alacritty](https://github.com/alacritty/alacritty) - recommended to use [tmux](https://github.com/tmux/tmux)/[i3](https://github.com/i3/i3) for splits and session's (The one I use)
+* [Kitty](https://github.com/kovidgoyal/kitty) - I never used it but people say it's nice
+* [Wezterm](https://github.com/wez/wezterm) - Lua configured terminal, I don't like the design especially how the cursor works in it, but it seems the community love it.
+
+### Nerdfont
+Its highly recommended to install [nerdfont](https://www.nerdfonts.com/) to support icons in your terminal, many plugins utilize it. \
+I use [CascadiaCode](https://www.programmingfonts.org/#cascadia-code), [JetBrainsMono](https://www.programmingfonts.org/#jetbrainsmono) is nice too.
+
+### Keybinds limitations
+Because of `ascii` terminals by default doesn't support Ctrl+Shift+X keys, [great comment that explains that](https://github.com/tmux/tmux/issues/674#issuecomment-263157843). \
```

Comment:
```suggestion
Because of `ascii` terminals don't support `Ctrl+Shift+X` keybindings, [here's a great comment that explains why](https://github.com/tmux/tmux/issues/674#issuecomment-263157843). \
```